setup_test_db.py

```diff
@@ -333,6 +333,31 @@
 ) ENGINE=InnoDB AUTO_INCREMENT=17 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 """
 
+create_cherrypicked_samples_view = """
```

Comment:
It's not great to store the view's definition in every place we need it to test, any chance to of using the container or is that too large a change? Worth another story?

Reply:
We could put it down as a new story but if there is going to be a new cherrypick service, it could be considered then?